crawler/emitter.py

```diff
@@ -241,6 +240,61 @@ def _publish_to_stdout(self):
                     print line.strip()
                     sys.stdout.flush()
 
+    def _publish_to_logstash(self, url, max_emit_retries=5):
```

Comment:
if we add new option, other targets (graphite) also need to implement that?

we can avoid new function in the code and use existing function by internally interpreting format. if target is http:// it's whole frame as a string, and if its logstash:// its json-per-feature.

in general, i think number of options to user be minimum for better usability.